backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -71,9 +82,30 @@ public async Task<ActionResult<LexAuthUser>> RegisterAccount(RegisterAccountInpu
         var user = new LexAuthUser(userEntity);
         await HttpContext.SignInAsync(user.GetPrincipal("Registration"),
             new AuthenticationProperties { IsPersistent = true });
+
+        var (jwt, _) = _lexAuthService.GenerateJwt(user);
```

Comment:
I don't think this will work, as `user` has `EmailVerificationRequired = true` we need to send a jwt via email that has that value changed first.

If you change the `LexAuthUser` from a `class` to a `record`, then you can simply do `GenerateJwt(user with {EmailVerificationRequired = false})` and it'll make a copy with that property changed.